shared-libs/rules-engine/test/rules-emitter.spec.js

```diff
@@ -214,7 +214,8 @@ describe('rules-emitter', () => {
 
       const { tasks, targets } = await rulesEmitter.getEmissionsFor([chtDocs.contact], [chtDocs.pregnancyReport]);
       expect(tasks.length).to.eq(1);
-      expect(targets.length).to.eq(1);
```

Comment:
This target isn't related to pregnancies.
<img width="440" alt="Screen Shot 2022-08-02 at 1 09 21 pm" src="https://user-images.githubusercontent.com/66472237/182304159-5666d00a-a19e-4381-9c87-f86943cdf09c.png">